helm-charts/common/tei/values.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Default values for tei.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+hftei:
+  volume: /mnt/model
+  shmSize: 1Gi
+
+image:
+  repository: ghcr.io/huggingface/text-embeddings-inference
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "cpu-1.2"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 6006
+  targetPort: 80
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+
+global:
```

Comment:
why set it to global here? 'global values' is usually for subchart to access the values from parent chart.

Reply:
> why set it to global here? Usually we leverage the helm's feature to override the sub chart's values in the parent chart, unless in some special case the sub chart needs to access the values of the parent chart, in that case we need to use 'global values'.

For user using parent chart to set the value, it will be easy to set all the values,like this:
  qnaConfig:
    REDIS_URL:  
    INDEX_NAME: 
    EMBEDDING_MODEL_ID: 
    RERANK_MODEL_ID:  
    LLM_MODEL_ID:  
    TEI_EMBEDDING_ENDPOINT:  
    TEI_RERANKING_ENDPOINT:  
    TGI_LLM_ENDPOINT:  
    MEGA_SERVICE_HOST_IP:  
    HUGGINGFACEHUB_API_TOKEN:  
    CHAT_BASE_URL:

Reply:
Is the above values under qnaConfig family is designed to be shared in multiple common charts, i.e. both common chart C1 and common chart C2 will share the same common global qnaConfig in a example workload? If so, I think we then should use global values.

Also, please refactor other issues besides this global one.